### ISSUE TITLE
Bug: Broken production deploy with mega mode

### ISSUE THREAD

**Issue by thepry** [tongueroo/jets#94]
Apparently deploy script is using not defined `fulL_cmd` variable for compiling rails assets.

https://github.com/tongueroo/jets/blob/9b8dd19b948e0b16494673dd6031ff9cd3a5a10f/lib/jets/builders/code_builder.rb#L224

```
Traceback (most recent call last):
	22: from /Users/mac/.rvm/gems/ruby-2.5.3/bin/ruby_executable_hooks:24:in `<main>'
	21: from /Users/mac/.rvm/gems/ruby-2.5.3/bin/ruby_executable_hooks:24:in `eval'
	20: from /Users/mac/.rvm/gems/ruby-2.5.3/bin/jets:23:in `<main>'
	19: from /Users/mac/.rvm/gems/ruby-2.5.3/bin/jets:23:in `load'
	18: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/jets-1.3.5/exe/jets:14:in `<top (required)>'
	17: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/jets-1.3.5/lib/jets/cli.rb:5:in `start'
	16: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/jets-1.3.5/lib/jets/cli.rb:21:in `start'
	15: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/jets-1.3.5/lib/jets/commands/base.rb:27:in `perform'
	14: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/jets-1.3.5/lib/jets/commands/base.rb:38:in `dispatch'
	13: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
	12: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
	11: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
	10: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/jets-1.3.5/lib/jets/commands/main.rb:21:in `deploy'
	 9: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/jets-1.3.5/lib/jets/commands/deploy.rb:29:in `run'
	 8: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/jets-1.3.5/lib/jets/commands/deploy.rb:50:in `build_code'
	 7: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/jets-1.3.5/lib/jets/commands/build.rb:26:in `build_code'
	 6: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/jets-1.3.5/lib/jets/builders/code_builder.rb:36:in `build'
	 5: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/jets-1.3.5/lib/jets/builders/code_builder.rb:215:in `compile_rails_assets'
	 4: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/bundler-1.17.2/lib/bundler.rb:313:in `with_clean_env'
	 3: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/bundler-1.17.2/lib/bundler.rb:562:in `with_env'
	 2: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/bundler-1.17.2/lib/bundler.rb:313:in `block in with_clean_env'
	 1: from /Users/mac/.rvm/gems/ruby-2.5.3/gems/jets-1.3.5/lib/jets/builders/code_builder.rb:216:in `block in compile_rails_assets'
/Users/mac/.rvm/gems/ruby-2.5.3/gems/jets-1.3.5/lib/jets/builders/code_builder.rb:224:in `rails_assets': undefined local variable or method `fulL_cmd' for #<Jets::Builders::CodeBuilder:0x00007f9ee1dc8638> (NameError)
```

**Comment by tongueroo** [comment]
Addressed in #96 Will go out shortly.

**Comment by tongueroo** [comment]
Will add this to the QA list.

Fixed and released in v1.3.6 [CHANGELOG](https://github.com/tongueroo/jets/blob/master/CHANGELOG.md)
